src/client.ts

```diff
@@ -115,7 +121,7 @@ export class SQLocal {
 
 		switch (message.type) {
 			case 'import':
-				this.worker.postMessage(
+				this.worker?.postMessage(
```

Comment:
The check at the top of the method makes optional chaining unnecessary here.

Reply:
good eyes. done